mypy/waiter.py

```diff
@@ -37,13 +37,11 @@ def __init__(self, name: str, args: List[str], *, cwd: str = None,
         self.passthrough = passthrough
 
     def start(self) -> None:
-        if self.passthrough is None:
+        if self.passthrough is None or self.passthrough < 0:
             self.outfile = tempfile.TemporaryFile()
         else:
             if self.passthrough >= 0:
```

Comment:
Never mind, `pytest -q` misbehaves in the presence of `xdist`, so I'm keeping the behavior as is.